notebooks/220-license-plate-recognition/README.md

```diff
@@ -0,0 +1,21 @@
+# License Plate Recognition with OpenVINO
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/zRAINj/openvino_notebooks/490f317a6d6015478612ff335f884f5c8a32bba2?urlpath=lab%2Ftree%2Fnotebooks%2F220-license-plate-recognition%2F220-license-plate-recognition.ipynb)
+
+License plate recognition model helps you get the chinese license plate number precisely in no time. The input of the color license plate image can be any size. It will be resized and augmented before put into the model. After matching the result to correct character, we can get the license plate number. In the notebook we show how to create the following pipeline:
+
+![text](https://user-images.githubusercontent.com/15709723/162659593-3f620d7a-44d2-4f49-a558-94c35a244a8e.png)
+
+And the result is:
+
+![text](https://user-images.githubusercontent.com/70456146/162759539-4a0a996f-dabe-40ea-98d6-85b4dce8511d.png)
+
+## Notebook Contents
+
+This notebook demonstrates license plate recognition with OpenVINO. We use the [License Plate Recognition Model](https://docs.openvino.ai/2020.2/_models_intel_license_plate_recognition_barrier_0001_description_license_plate_recognition_barrier_0001.html) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). This model uses a small-footprint network trained end-to-end to recognize Chinese license plates in traffic.
```

Comment:
Please use the new link for the model https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/license-plate-recognition-barrier-0001